edx_sga/tests.py

```diff
@@ -140,7 +140,22 @@ def test_max_score_integer(self):
     @mock.patch('edx_sga.sga.render_template')
     @mock.patch('edx_sga.sga.Fragment')
     def test_student_view(self, Fragment, render_template):
-        block = self.make_one()
+        def make_one(**kw):
```

Comment:
Instead of copying this entire function, you should just make the display name a passable argument in the original function with a default value of `None` and have a conditional that checks if it is set before settings the display name of the block